CONTRIBUTING.rst

```diff
@@ -333,6 +332,60 @@ Recommendation for the Pull Requests:
    -  If applicable, ensure that your fix includes appropriate tests. Adding tests for your bug fix helps prevent future regressions and maintains the stability of the software.
 
 
+Contributing to test data
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If your changes include updating test data, you may work with the ``gpm/tests/data/`` directory as you would with any other git repository.
+You need to ask the maintainers to become a contributor on the `gpm_api_test_data <https://github.com/ghiggi/gpm_api_test_data>`_ repository to create a branch and a Pull Request.
```

Comment:
```suggestion
To do this, you need to ask the maintainers to become a contributor on the `gpm_api_test_data <https://github.com/ghiggi/gpm_api_test_data>`_ repository in order that you can create a branch and a Pull Request.
Then you will able to modify the ``gpm/tests/data/`` directory/repository as you would with any other git repository.
```

Reply:
Is it necessary that someone ask to become a contributor? He can't just fork and PR?

Reply:
@ghiggi well if you make a fork of the test-data repo, you need to update the submodule url in the gpm-api repo (and restore it before merging the pr), which imo complexify even more the process.

Git LFS would have been an elegant alternative solution but bandwidth quotas are absurly low :(

Reply:
Ah right. Didn't thought about that ...  
Just merge the suggestion and I merge the PR @sphamba ;)